lib/Kconfig

```diff
@@ -34,4 +34,7 @@ source "lib/utils/Kconfig"
 source "lib/uuid/Kconfig"
 
 source "lib/min_heap/Kconfig"
+
+source "lib/libmp/Kconfig"
```

Comment:
Can it be shortened to "mp" instead of "libmp"?
It's placed in the "lib" folder, so it's clear that it's a library.